www/assets/app.css

```diff
@@ -55,26 +54,42 @@ table {
 
 /* Header */
 body > header {
-  padding: 15px 0 35px;
-}
-body > header:after {
-  content: '';
-  display: block;
-  clear: both;
+  padding: 25px 0;
+  position: relative;
 }
 body > header h1 {
-  float: left;
   font-size: 2em;
+  line-height: 1em;
   margin: 0;
 }
+@media (min-width: 768px) {
+  body > header {
+    padding: 45px 0;
+  }
+  body > header h1 {
+    position: absolute;
+    left: 0;
+    top: 50%;
+    transform: translateY(-50%);
+  }
+  body > header nav {
+    position: absolute;
+    right: 0;
+    top: 50%;
+    transform: translateY(-50%);
```

Comment:
Having trouble getting the page to load (it can't find hoodie), will try to look soon though sorry.